wrapanapi/systems/rhevm.py

```diff
@@ -66,20 +68,37 @@ def _get_network(self, network_name):
         except IndexError:
             raise ItemNotFound('No match for network by "name={}"'.format(network_name))
 
-    def update_nic(self, network_name, nic_name='nic1', interface='VIRTIO'):
-        nic_service = self._get_nic_service(nic_name)
-        nic = nic_service.get()
+    def get_nics(self):
+        return self.api.nics_service().list()
+
+    def update_nic(self, network_name, nic_name='nic1', interface='VIRTIO', on_boot=True,
+                   add_nic=False, vnic_profile=None):
+        """Update or add a nic on VM/Template
+        Args:
+            network_name: string name of the network, also default for vnic_profile name if empty
+            nic_name: string name of the nic to add
+            interface: string interface type for pyVmomi, interfaces are resolved to a specific type
+            on_boot: boolean, kwarg for nic options
+            add_nic: boolean, if true will add a new nic instead of looking up existing one
```

Comment:
*OPTIONAL*

What if someone tries to add a nic that matches an existing nic name. I'd like to see `add_nic` and `update_nic` as two different methods. `update_nic` always expects it to exist and throws an error if it does not. `add_nic` expects it to not already exist, and raises an error if it does already exist, or perhaps has a `overwrite=True` kwarg. (the 2 methods could end up sharing common code from a `_private_method`)